content/rs/installing-upgrading/upgrading.md

```diff
@@ -7,34 +7,87 @@ alwaysopen: false
+    - To upgrade your cluster to v6.2, your cluster must first be on 6.0 or above _and_ your database nodes must be running Redis 6.
 
-- To upgrade your cluster to v6.0, your cluster must first be on 5.4.0 or above
-    and the databases must be running Redis 5.
     - To upgrade your cluster to v6.0.20, your cluster must first be on 5.6.0
-- To upgrade your cluster to v5.6, your cluster must first be on 5.0.2-30 or above.
-- To upgrade your cluster to v5.4, your cluster must first be on 5.0 or above.
-- To upgrade your cluster to v5.2, your cluster must first be on 4.5 or above.
 
-The upgrade process for a Redis Enterprise Software cluster is "ongoing" when the nodes in the cluster have mixed versions.
-The upgrade is only considered complete when all of the nodes are upgraded to the new version.
+    - To upgrade your cluster to v6.0, your cluster must first be on 5.4.0 or above _and_ the databases must be running Redis 5.
 
-{{< warning >}}
+    - To upgrade your cluster to v5.6, your cluster must first be on 5.0.2-30 or above.
 
-Using features from the newer version before all nodes are upgraded can produce unexpected results or cause failures in the cluster.
+    - To upgrade your cluster to v5.4, your cluster must first be on 5.0 or above.
 
-{{< /warning >}}
+    - To upgrade your cluster to v5.2, your cluster must first be on 4.5 or above.
+
+3.  Avoid using features introduced in the newer version until after all nodes have been upgraded; otherwise, unexpected results or cluster failures can occur.
+
+## Upgrade process
+
+The upgrade process includes several tasks, including upgrading:
+
+- [Open source Redis](#upgrade-open-source-redis)
+- [Each node](#upgrade-a-node) in the cluster
+- [Each database](#upgrade-a-database)
+- [Active-active databases](#upgrade-activeactive-databases)
+
+The following sections provide more info.
+
+The upgrade process for a Redis Enterprise Software cluster is considered _ongoing_ when nodes in the cluster have mixed versions.
+
+The upgrade process is only considered _complete_ when all nodes are upgraded to the new version.
+
+### Upgrade open source Redis
+
+Part of the Redis Enterprise Software upgrade process includes upgrading open source (OSS) Redis, also known as _core_ Redis.  
+
+Each upgrade of Redis Enterprise Software includes _two_ versions of OSS Redis to help smooth the upgrade process:
+
+- The _latest_ release reflects the most recent minor release (example: Redis 6.2), whether major (x.0 release) or minor (x.y).
+
+- The last _major_ release reflects the most recent major release (example: 6.0).
+
+When you upgrade OSS Redis, you can upgrade from:
+
+- The previous major release to the current major release.
 
-## Upgrade a node
+- The previous minor release to the current minor release.
+
+If you're not currently using either of supported previous releases, you need to separately update OSS Redis to one of those releases; otherwise, the update to Redis Enterprise Software fails.
```

Comment:
```suggestion
If you're not currently using either of the supported previous releases, you need to separately update OSS Redis to one of those releases; otherwise, the update to Redis Enterprise Software fails.
```

Reply:
Fixed.